netsim/modules/lag.py

```diff
@@ -355,6 +355,26 @@ def populate_mlag_peer(node: Box, intf: Box, topology: Box) -> None:
   
   intf.pop('vlan',None)                                                     # Remove any VLANs provisioned on peerlinks
 
+def check_bridge_links(topology: Box) -> None:
+  err_cache: dict = {}
+  for link in topology.links:
+    if '_virtual_lag' not in link:
+      continue
+
+    if 'bridge' not in link:
+      continue
+
+    nodes = sorted([ intf.node for intf in link.interfaces ])
+    n_text = " and ".join(nodes)
+    if n_text in err_cache:
+      continue
+
+    log.warning(
```

Comment:
Need to check if LACP is active - LACP off will work fine

Reply:
That's why the warning is saying "LACP will not work". I didn't have the willpower to dig through the data structures to figure out if LACP is configured on that link (the information is not exactly readily available).

I don't think it's worth spending more time on this. Anyone using LAG without LACP between a VM and a container can just turn off the warning 😜